src/components/AlertesControles/AlertesControles.tsx

```diff
@@ -13,9 +13,16 @@ export function AlertesControles(props: OrchestratedElement) {
 			);
 		});
 		return (
-			<div className={fr.cx('fr-alert', type)}>
-				<h3 className={fr.cx('fr-alert__title')}>Il y a un problème</h3>
-				{content}
+			<div className="fr-grid-row fr-grid-row--center fr-grid-row--middle fr-mt-6w fr-mt-md-12w ">
+				<div
+					className="fr-col-lg-6 fr-col-12"
+					style={{ marginBottom: fr.spacing('4v') }}
```

Comment:
we can also use the class `fr-mb-4w ` to add the margin bottom. Let me know what's best in your opinion.

Reply:
Is there a reason why we don't use the react-dsfr componant here?

Reply:
il y a un composant dsfr pour la grille ?

Reply:
Pardon, je n'était pas clair -- je parlais plutot du composant `<Alert />` qui exist dans React-DSFR